src/main/java/stringaddcalculator/Nums.java

```diff
@@ -0,0 +1,48 @@
+package stringaddcalculator;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Nums {
+
+    private final List<Integer> nums;
+
+    private Nums(List<Integer> nums) {
+        this.nums = nums;
+    }
+
+    public static Nums of(String[] splitNumsString) {
+        List<Integer> numList = new ArrayList<>();
+        for (String numString : splitNumsString) {
+            int num = Integer.parseInt(numString);
+            if (num < 0) {
+                throw new RuntimeException("숫자는 0 이상이어야 합니다.");
+            }
+            numList.add(num);
+        }
+        return new Nums(numList);
+    }
+
+    public int sum() {
+        return nums.stream().mapToInt(num -> num).sum();
+    }
```

Comment:
stream 이후로는 개행을 해주면 좀 더 가독성이 좋아질 것 같은데 어떻게 생각하시나요? 😄 
```suggestion
    public int sum() {
        return nums.stream()
                .mapToInt(num -> num)
                .sum();
    }
```